src/server/pachyderm_test.go

```diff
@@ -7935,3 +8003,22 @@ func getPachClient(t testing.TB) *client.APIClient {
 	})
 	return pachClient
 }
+
+var etcdClient *etcd.Client
+var getEtcdClientOnce sync.Once
+
+const (
+	etcdAddress = "localhost:32379" // etcd must already be serving at this address
```

Comment:
Grepping the code base, it looks like we're providing this address in a bunch of test suites, while it's defined in the assets package, and the driver hardcodes the port (and assumes its there) as well. Seems ripe for a const definition someplace to keep this DRYer, unless I'm missing something.

Reply:
So this actually wound up being harder than I thought, I changed the code but where to put the constant is a bit tricky due to import dependency cycles. I decided to just create an issue for this and not block the PR. Since this isn't a new problem introduced by the PR but rather one we already had.

Reply:
:+1: